iocore/net/UnixNetAccept.cc

```diff
@@ -45,10 +43,14 @@ net_accept(NetAccept *na, void *ep, bool blockable)
   Event *e               = static_cast<Event *>(ep);
   int res                = 0;
   int count              = 0;
-  const int loop         = NetAccept::accept_till_done;
   UnixNetVConnection *vc = nullptr;
   Connection con;
 
+  EThread *t             = e->ethread;
+  NetHandler *handler    = get_NetHandler(t);
+  int config_value       = handler->config.additional_accepts;
```

Comment:
If you change this to `... + 1`, then can use use `< instead of `<=` in the loop condition and avoid having to add the "+1" in multiple locations?